exchanges/cryptodotcom/cryptodotcom.go

```diff
@@ -835,7 +752,7 @@ func (cr *Cryptodotcom) getParamString(params map[string]interface{}) string {
 		case reflect.Map:
 			param, ok := params[keys[x]].(map[string]interface{})
 			if !ok {
-				param = map[string]interface{}{}
+				param = make(map[string]interface{})
```

Comment:
In another PR I have suggested we use the switch case to access an interfaces underlying inbuilt type. We can use that here instead of using reflection which can be slow in some instances.